betting-game-react/src/App.js

```diff
@@ -1,12 +1,16 @@
-import React from 'react'
-import './App.css'
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import './App.css';
+import Navbar from './components/Navbar/Navbar';
 
 function App() {
   return (
-    <div className='min-h-screen min-w-screen bg-gray-900 text-center text-6xl text-gray-200 grid items-center tracking-wider'>
-      AWESOME BETTING APP
+    <div>
+      <Router>
+        <Navbar></Navbar>
+      </Router>
     </div>
-  )
+  );
 }
 
-export default App
+export default App;
```

Comment:
I think if you have only one function - you can actually write 
`export default function App() {...`
:)

Reply:
Could you also change the App.js to an App.tsx?

Reply:
Done :-)